src/Main.js

```diff
@@ -1,45 +1,36 @@
 import React, { Component } from 'react'
-import HornedBeast from './HornedBeasts';
+import Data from './Data.json'
+import HornedBeast from './HornedBeasts'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
 
 class Main extends Component {
-
-  hornedBeastArray = [{
-    "image_url": "http://3.bp.blogspot.com/_DBYF1AdFaHw/TE-f0cDQ24I/AAAAAAAACZg/l-FdTZ6M7z8/s1600/Unicorn_and_Narwhal_by_dinglehopper.jpg",
-    "title": "UniWhal",
-    "description": "A unicorn and a narwhal nuzzling their horns",
-    "keyword": "narwhal",
-    "horns": 1
-  },
-
-  {
-    "image_url": "https://images.unsplash.com/photo-1512636618879-bbe79107e9e3?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=bd9460ee6d1ddbb6b1ca7be86dfc4590&auto=format&fit=crop&w=1825&q=80",
-    "title": "Rhino Family",
-    "description": "Parent rhino with two babies",
-    "keyword": "rhino",
-    "horns": 2
-  },
   
-  {
-    "image_url": "https://www.dhresource.com/0x0s/f2-albu-g5-M00-1A-11-rBVaI1hsIIiALxKzAAIHjSU3VkE490.jpg/wholesale-halloween-costume-prop-unicorn.jpg",
-    "title": "Unicorn Head",
-    "description": "Someone wearing a very silly unicorn head mask",
-    "keyword": "unicorn", 
-    "horns": 1
-  }];
-
-  newArray = [];
+  constructor(props) {
+    super(props);
+    this.state = {
+      beastData: [],
+    }
+
+    Data.forEach( (element, index) => 
+    this.state.beastData.push(
+      <Col  xs={4} md={2}>
+      <HornedBeast 
+      title={element.title} 
+      src={element.image_url} 
+      description={element.description} 
+      key={index}/>
+    </Col>
+      ));

```

Comment:
- For this section, I would try and use map(). If you time, look into why we use map vs forEach in React 👍 
- Indentation is needed on lines 19-22  👍 
- Change capital `Data` to lowercase `data` since data is not a component 👍